src/utils/dif_upload.rs

```diff
@@ -775,9 +800,13 @@ fn collect_auxdif<'a>(
         .file_stem()
         .map(|stem| stem.to_string_lossy())
         .unwrap_or_default();
+
+    // TODO(Nar): Can't seem to get past this parse call
```

Comment:
It's trying to parse the path of the file using https://github.com/getsentry/rust-debugid which is not implemented for dartsymbols. But from what I see, you shouldnt go through the `auxdif` at all, as you'll be collecting those files directly. This however, requires you to first implement a new format at https://github.com/getsentry/symbolic/blob/master/symbolic-debuginfo/src/lib.rs